HarClient.Tests/Validation/HarAssert.cs

```diff
@@ -0,0 +1,251 @@
+﻿using HarSharp;
+using Microsoft.VisualBasic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace JBlam.HarClient.Tests.Validation
+{
+    static class HarAssert
```

Comment:
Consider renaming to indicate that this is a class about HAR JSON format, rather than e.g. "is the content of the HAR internally consistent" which could be done more effectively against the `HarSharp.Har` model.

Reply:
Rejected. Public API deals entirely with the data model object.